ai_worker/src/service_rag/legal_parser.py

```
@@ -0,0 +1,312 @@
+            text: 법령 조문 텍스트
+            statute_id: 법령 고유 ID (없으면 UUID 생성)
+            metadata: 추가 메타데이터
+
+        Returns:
+            Statute: 파싱된 법령 객체
+
+        Raises:
+            ValueError: 빈 텍스트인 경우
+        """
+        if not text or text.strip() == "":
+            raise ValueError("Empty statute text")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)